cmd/create/cluster/flag.go

```diff
@@ -28,7 +28,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&f.Config, flagConfig, "g", "", `The path to the file containing API endpoints and tokens for each provider.`)
 	cmd.Flags().StringVarP(&f.Kubeconfig, flagKubeconfig, "k", "", `The path to the directory containing the kubeconfigs for provider control planes.`)
 	cmd.Flags().StringVar(&f.Output, flagOutput, "", `The directory in which to store the cluster ID, kubeconfig, and provider of the created cluster.`)
-	cmd.Flags().StringVarP(&f.Installation, flagInstallation, "i", "", `The target management cluster type to be used ('aws', 'azure', 'kvm', or 'aws-china').`)
+	cmd.Flags().StringVarP(&f.Installation, flagInstallation, "i", "", `The target management cluster type to be used ('aws', 'azure', 'kvm', 'gce', 'openstack' or 'aws-china').`)
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&f.Installation, flagInstallation, "i", "", `The target management cluster type to be used ('aws', 'azure', 'kvm', 'gcp', 'openstack' or 'aws-china').`)
```
In the map of features you are using `gcp` as map key. `gcp` is also the name we use in other places for the google provider.

Reply:
thx, was done by accident. fixed